hack/machineset.sh

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env bash
+
+# machineset.sh - create a Windows MachineSet for Windows Machine Config Operator
+# Creates a MachineSet.yaml file and apply/delete the MachineSet if optional action is provided
+#
+# USAGE
+#    machineset.sh
+# OPTIONS
+#    $1      Action       (Optional) apply/delete the MachineSet
+# PREREQUISITES
+#    oc                   to fetch cluster info and apply/delete MachineSets on the cluster(cluster should be logged in)
+#    aws                  to fetch Windows AMI id for AWS platform (only required for clusters running on AWS)
+set -euo pipefail
+
+ACTION=${1:-}
+
+function error-exit() {
+    echo "Error: $*" >&2
+    exit 1
+}
+
+# get_spec returns the template yaml common for all cloud providers
+get_spec(){
```

Comment:
Space before `{`. Comment applies to `get_aws_ms` and `get_azure_ms` also.

Nit: You ideally want a guard that the helper is called with expected number of inputs.